clients/drcachesim/common/trace_entry.h

```diff
@@ -747,6 +747,123 @@ struct schedule_entry_t {
     uint64_t instr_count;
 } END_PACKED_STRUCTURE;
 
+#ifdef BUILD_PT_TRACER
+
+/**
+ * The type of a syscall PT entry in the offline output.
+ */
+typedef enum {
+    SYSCALL_PT_ENTRY_TYPE_PID = 0,
+    SYSCALL_PT_ENTRY_TYPE_THREAD,
+    SYSCALL_PT_ENTRY_TYPE_PT_METADATA_BOUNDARY,
```

Comment:
Add a short description where needed, particularly for the boundary ones.

Reply:
Done.